### PR TITLE
docs: Add ruff section in code style page

### PR DESCRIPTION
**Pull request by Batalex**
I think we should add basic configurations for flake8 and isort to be compatible with black.

- Isort is easy: `profile = black`
- Flake8 is trickier: rules E203, E501 and W503 need to be excluded

### CODE REVIEW

File: package-structure-code/code-style-linting-format.md [hunk 1]
```diff
@@ -6,10 +6,11 @@
 
 * pyOpenSci requires authors to follow PEP 8 code format guidelines
 * Setting up a code formatters like Black and isort will help you enforce PEP 8 style guidelines and also consistent, readable code format
-* Some commonly used tools are: Black/Blue, Isort, flake8
+* Some commonly used tools are: Black/Blue, Isort, flake8, ruff
```
Comment (lwasser) [inline package-structure-code/code-style-linting-format.md:9]:
```suggestion
* Some commonly used tools are: Black/Blue, Isort, flake8, Ruff
```

Comment (lwasser) [reply package-structure-code/code-style-linting-format.md:9]: @Batalex what do you think about removing blue? i was just thinking it's one more option that will confuse folks and why not just stick with black to keep things simple.

File: package-structure-code/code-style-linting-format.md [hunk 2]
```diff
@@ -92,7 +94,7 @@ some exceptions. A few examples of those exceptions are below:
 
 - Black defaults to a line length of 88 (79 + 10%) rather than the 79 character `PEP 8` specification. However, line length is a setting can be manually overwritten in your Black configuration.
 - Black and Blue will not adjust line length in your comments or docstrings.
```
Comment (lwasser) [inline package-structure-code/code-style-linting-format.md:96]: if we do this - we'd them just remove blue everywhere and add Ruff as you already have!

**Review by lwasser — CHANGES_REQUESTED** [review]
@Batalex this looks awesome! i made a few comments that are very minor - my thought it to just remove blue and add Ruff here to keep decision making simple. :) but let me know what your thoughts are!

File: package-structure-code/code-style-linting-format.md [hunk 3]
```diff
@@ -197,6 +199,46 @@ from stravalib import exc
 from stravalib import unithelper as uh
 ```
 
+### ruff
```
[the remark this answers is not in the document]
Comment (Batalex) [reply package-structure-code/code-style-linting-format.md]: Done in fa6a15b98f850047e3b5f18df50a89aa03e531b8.

> i know this is super nitpicky and minor

I agree that consistency is essential, as well as following authors' directive on how to name things!

File: package-structure-code/code-style-linting-format.md (hunk 2 above)
Comment (Batalex) [reply package-structure-code/code-style-linting-format.md]: Removed blue in f39597e61b5f3940a1d62e5aaca3f0bee46d818b, but quite easy to revert

File: package-structure-code/code-style-linting-format.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (lwasser) [reply package-structure-code/code-style-linting-format.md]: wonderful! thank you!!

File: package-structure-code/code-style-linting-format.md [hunk 4]
```diff
@@ -84,15 +86,15 @@ You will learn more about Black below.
 
 ### Black and Blue
```
Comment (lwasser) [inline package-structure-code/code-style-linting-format.md:87]:
```suggestion
### Black
```

File: package-structure-code/code-style-linting-format.md (hunk 1 above)
Comment (lwasser) [reply package-structure-code/code-style-linting-format.md]: excellent!! i've suggested a few other sections we can remove as well on blue only :) then we can merge this!

**Comment by lwasser** [comment]
@Batalex how do you feel about having more superpowers for this repo? so you could merge and review prs and such?

File: package-structure-code/code-style-linting-format.md (hunk 2 above)
Comment (Batalex) [reply package-structure-code/code-style-linting-format.md]: Oups, I missed the capitalized "Blue"

**Comment by Batalex** [comment]
> @Batalex how do you feel about having more superpowers for this repo? so you could merge and review prs and such?

Yeaaay, **unlimited power** ⚡

**Comment by lwasser** [comment]
only failing build because of a 500 error for conda-forge - the link works. so we can merge

**Pull request merged**